packages/visualizations/src/components/Table/Header.svelte

```diff
@@ -1,14 +1,20 @@
 <script lang="ts">
-    import type { Column } from './types';
+    import { ColumnSort, type Column } from './types';
 
     export let columns: Column[];
 </script>
 
 <thead>
     <tr>
         {#each columns as column}
-            <th class={`table-header--${column.dataFormat}`}>
+            <th class={`table-header--${column.dataFormat}`} on:click={column?.onClick}>
```

Comment:
`th` nodes are not accessible via keyboard navigation (at least I couldn't during my tests, maybe there is keyboard shortcut that I don't know), we can't trigger a click from them when pressing enter. 

Even if we have no design for the sort system, I think we should consider adding this mechanism from the start

Reply:
A good example: https://www.w3.org/WAI/ARIA/apg/patterns/table/examples/sortable-table/